src/components/TrainModelFirstTitle.svelte

```diff
@@ -18,15 +21,20 @@
     {$t('content.model.trainModelFirstHeading')}
   </h1>
   <p class="mb-5">
-    {$t('content.model.trainModelBody1')}
-  </p>
-  <p class="mb-10">
-    {$t('content.model.trainModelBody2')}
+    {$t('content.model.trainModelBody')}
   </p>
-  <div class="flex gap-5">
-    <StandardButton onClick={() => navigate(Paths.DATA)}>
-      {$t('content.model.addData')}
-    </StandardButton>
-    <TrainingButton type="secondary" onClick={() => navigate(Paths.TRAINING)} />
+  {#if !sufficientData}
+    <p>
+      {$t('content.model.notEnoughDataInfoBody')}
+    </p>
+  {/if}
```

Comment:
Can we wrap these paragraphs in a div with space-y-5 and lose the mb-5? With sufficientData we only show the first paragraph. For some reason the margin doesn't collapse so we get a lot of vertical space between text and button.

![image](https://github.com/microbit-foundation/ml-trainer/assets/95928279/29e8da63-88ee-4388-a902-f56d74ceca2c)

Reply:
Makes sense!

Reply:
[10b2dd9](https://github.com/microbit-foundation/ml-trainer/pull/135/commits/10b2dd95234b6fc94bb36a1134645e8ac60a9c97)